<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWindow</class>
 <widget class="QWidget" name="DepositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>910</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_toMain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Limelight</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #16082f; border: none; border-radius: 15px;
color: white;
}
                         QPushButton:hover {
                           background-color: #18174f; /* Цвет фона при наведении */
                         }
                         QPushButton:pressed {
                         background-color: #18174ffe; /* Цвет фона при нажатии */
color: grey;
                        }
</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>891</width>
     <height>591</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_totalS">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сумма вклада</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_totalS">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2" rowspan="9">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_total_percent">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Начисленные проценты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total_percent_res">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
qproperty-alignment: 'AlignCenter';
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total_tax">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сумма налога</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total_tax_res">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
qproperty-alignment: 'AlignCenter';
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сумма на вкладе</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total_res">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
qproperty-alignment: 'AlignCenter';
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_term">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Срок вклада</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_term">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_term">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Месяцев</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Лет</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_percent">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Процентная ставка (%)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_percent">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_percent_tax">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Налоговая ставка (%)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_percent_tax">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_capitalization">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Капитализация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_capitalization">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_periodicity_cap">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Периодичность выплат</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_cap">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Раз в месяц</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в два месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в три месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в четыре месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в шесть месяцев</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в год</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_periodicity_rep">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Периодичность пополнения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_periodicity_rep">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Не предусмотрено</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в месяц</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в два месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в три месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в четыре месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в шесть месяцев</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в год</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_replenishment">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Пополнение вклада</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_replenishment">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_periodicity_wit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Периодичность частичных снятий</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_periodicity_wit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Не предусмотрено</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в месяц</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в два месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в три месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в четыре месяца</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в шесть месяцев</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Раз в год</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0" rowspan="2" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_withdrawal">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выплата</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_withdrawal">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Limelight</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="pushButton_calculate">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Limelight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #16082f; border: none; border-radius: 15px;
color: white;
}
                         QPushButton:hover {
                           background-color: #18174f; /* Цвет фона при наведении */
                         }
                         QPushButton:pressed {
                         background-color: #18174ffe; /* Цвет фона при нажатии */
color: grey;
                        }
</string>
      </property>
      <property name="text">
       <string>Рассчитать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
